tests/script/test_shared_id.py

```diff
@@ -166,7 +167,13 @@ async def test_sharedId_in_different_navigable(websocket, context_id, html):
 
     await goto_url(websocket, context_id, html("some other page"))
 
-    with pytest.raises(Exception) as exception_info:
+    with pytest.raises(
+            Exception,
+            match=re.compile(
+                str({
+                    "error": "no such node",
+                    "message": 'SharedId ".*" belongs to different document. Current document is .*'
+                }))):
```

Comment:
I'd prefer to have a more readable verification here. What about the following?

```suggestion
            check=lambda e: {
                "error": "no such node",
                "message": AnyContains("different document")
            } == exception_info.value.args[0]
        )
```

Reply:
Where did you get the `check` argument from?

https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-raises

It doesn't exist

Reply:
The only way to do what you're suggesting is to revert to how it was before whenever a regex is involved.

Reply:
Indeed it's not in the API, my bad